### PR TITLE
build 0.6.5

### PR DESCRIPTION
**Pull request by boldorider4**
* [Jira Ticket](https://anaconda.atlassian.net/browse/PKG-3179)
* [Upstream project](https://github.com/defunkt/pystache)
* dependency for [rbcl](https://anaconda.atlassian.net/browse/PKG-3088)

Notes:
* indentation has been fixed from 4 to 2 spaces.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,52 +1,60 @@
...
-    build:
-        - python
-        - setuptools
-
-    run:
-        - python
+  host:
+    - python
+    - setuptools
+    - setuptools-scm 7.1.0
```
Comment (skupr-anaconda) [inline recipe/meta.yaml]:
```suggestion
    - setuptools-scm
```
To simplify python 3.12 build-out

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,52 +1,60 @@
...
+    - wheel
+    - pip
+  run:
+    - python
+    - importlib-metadata >=4.6
```
Comment (skupr-anaconda) [inline recipe/meta.yaml]:
```suggestion
    - importlib-metadata >=4.6  # [py<310]
```
See https://files.pythonhosted.org/packages/4d/6f/e3d8ec5c636dc69a6dadd27674addc8abf5627c927d3cbcd853b3ba9093d/pystache-0.6.5.tar.gz

File: recipe/meta.yaml (hunk 1 above)
Comment (boldorider4) [reply recipe/meta.yaml]: Thank you!

File: recipe/meta.yaml (hunk 2 above)
Comment (boldorider4) [reply recipe/meta.yaml]: Thank you!

**Review by lorepirri — APPROVED** [review]

**Review by skupr-anaconda — APPROVED** [review]

**Pull request merged**